collectors/charts.d.plugin/speedtest/speedtest.chart.sh

```diff
@@ -0,0 +1,160 @@
+# shellcheck shell=bash
+# no need for shebang - this file is loaded from charts.d.plugin
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# netdata
+# real-time performance and health monitoring, done right!
+# (C) 2016 Costa Tsaousis <costa@tsaousis.gr>
+#
+
+# if this chart is called X.chart.sh, then all functions and global variables
+# must start with X_
+
+# _update_every is a special variable - it holds the number of seconds
+# between the calls of the _update() function
+speedtest_update_every=120
```

Comment:
Well you set the default 120 seconds (2 minutes) , is it not too high?

Reply:
Reasons I set the default to 120 seconds:
- Each speedtest run takes roughly 20-30 seconds to complete. 
- Each speedtest downloads and uploads significant amount of bytes.

Reply:
Thanks!